lib/Jsrt/ChakraCore.h

```diff
@@ -679,5 +679,42 @@ CHAKRA_API
         _In_ JsPropertyIdRef propertyId,
         _Out_ bool *hasOwnProperty);
 
+/// <summary>
+///     Write JS string value into char string buffer without a null terminator
```

Comment:
Can you add some more details about the semantics of this API, specifically what happens if the codepoint being copied is not within the char range